src/researches/relevantWords.js

```diff
@@ -23,9 +32,16 @@ function relevantWords( paper ) {
 		subheadings,
 	};
 
-	const relevantWordsFromPaperAttributes = getRelevantWordsFromPaperAttributes( attributes, locale );
+	const relevantWordsFromPaperAttributes = getRelevantWordsFromPaperAttributes( attributes, language, morphologyData );
+
+	/* If a word is used in any of the attributes, its relevance is automatically high.
```

Comment:
Yay, explanation of the magic number 👍 


We have a code style for multiline comments:

```
/*
 * Comment here.
 * Another line here.
 */
```

Reply:
👍